modules/home-manager/wezterm.nix

```
@@ -0,0 +1,26 @@
+{ catppuccinLib }:
+{ config, lib, ... }:
+
+let
+  inherit (config.catppuccin) sources;
+  cfg = config.catppuccin.wezterm;
+in
+
+{
+  options.catppuccin.wezterm = catppuccinLib.mkCatppuccinOption { name = "wezterm"; };
+
+
+  config = lib.mkIf cfg.enable {
+    programs.wezterm = {
+      colorSchemes = lib.importTOML "${sources.wezterm}/dist/catppuccin-${cfg.flavor}.toml";
```

I just noticed its a attrs of toml's
```suggestion
      colorSchemes."catppuccin-${cfg.flavour}" = lib.importTOML "${sources.wezterm}/dist/catppuccin-${cfg.flavor}.toml";
```